src/modules/twap/utils/parseTwapError.ts

```diff
@@ -0,0 +1,26 @@
+enum TwapErrorCodes {
+  INVALID_ARGUMENT = 'INVALID_ARGUMENT',
+}
+
+const TwapErrorMessages = {
+  DEFAULT: 'Something went wrong creating your order',
+}
+
+function parseInvalidArgumentError(error: any) {
+  const regex = /(?<=\(argument=")[^"]+(?=",)/g
+  const matches = error.message.match(regex)
+  const invalidArgument = matches?.length ? matches[1] : ''
+  const str = invalidArgument ? `"${invalidArgument}" ` : ''
+
+  return `Invalid argument ${str}provided`
+}
+
+export function parseTwapErrorMessage(error: any) {
```

Comment:
I'm not sure that we should separately consider INVALID_ARGUMENT error.
More likely, we should generally derive an error message, like:
```
return error.message || error.toString()
```

Reply:
The reason for this was to have user friendly error message instead of something like this 
![Screenshot 2023-07-19 at 17 42 02](https://github.com/cowprotocol/cowswap/assets/34926005/9f062d54-c166-4a29-adf9-1f5b19eda739)